source/fundamentals/builders/filters.txt

```diff
@@ -11,3 +11,571 @@ Filters Builders
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Geospatial Method 
+     - Matches
+      
+   * - :java-core-api:`geoWithin() <com/mongodb/client/model/Filters.html#geoWithin(java.lang.String,org.bson.conversions.Bson)>`
+     - documents containing a GeoJSON geometry value that falls within a bounding GeoJSON geometry.
+
+   * - :java-core-api:`geoWithinBox() <com/mongodb/client/model/Filters.html#geoWithinBox(java.lang.String,double,double,double,double)>`
+     - documents containing a coordinates value that exist within the specified box.
+      
+   * - :java-core-api:`geoWithinPolygon() <com/mongodb/client/model/Filters.html#geoWithinPolygon(java.lang.String,java.util.List)>`
+     - documents containing a coordinates value that exist within the specified polygon.
+
+   * - :java-core-api:`geoWithinCenter() <com/mongodb/client/model/Filters.html#geoWithinCenter(java.lang.String,double,double,double)>`
+     - documents containing a coordinates value that exist within the specified circle.
+
+   * - :java-core-api:`geoWithinCenterSphere() <com/mongodb/client/model/Filters.html#geoWithinCenterSphere(java.lang.String,double,double,double)>`
+     - geometries containing a geospatial data value (GeoJSON or legacy coordinate pairs) that exist within the specified circle, using spherical geometry.
+
+   * - :java-core-api:`geoIntersects() <com/mongodb/client/model/Filters.html#geoIntersects(java.lang.String,org.bson.conversions.Bson)>`
+     - geometries that intersect with a GeoJSON geometry. The 2dsphere index supports $geoIntersects.
+
+   * - :java-core-api:`near() <com/mongodb/client/model/Filters.html#near(java.lang.String,org.bson.conversions.Bson,java.lang.Double,java.lang.Double)>`
+     - geospatial objects in proximity to a point. Requires a geospatial index. The 2dsphere and 2d indexes support $near.
+
+   * - :java-core-api:`nearSphere() <com/mongodb/client/model/Filters.html#nearSphere(java.lang.String,org.bson.conversions.Bson,java.lang.Double,java.lang.Double)>`
+     - geospatial objects in proximity to a point on a sphere. Requires a geospatial index. The 2dsphere and 2d indexes support $nearSphere.
+
+
+The following example creates a filter that matches documents in which
+the ``point`` field contains a GeoJSON geometry that falls within
+the given polygon in the ``geo_points`` collection:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/Filters.java
+   :language: java
+   :dedent:
+   :start-after: begin geoWithinComparison
+   :end-before: end geoWithinComparison
+
+The JSON output renders as the following:
+
+.. code-block:: json
+      :copyable: false
+   
+   {
+      "point": {
+         $geoWithin: {
+            $geometry: {
+               type: "Polygon",
+               coordinates: [ [[ 0, 0], [4 , 0], [4, 4], [0, 4], [0 , 0] ] ]
+            }
+         }
+      }
+   }
+
+The output of the above code snippet should look something like this:
```

Comment:
Nitpick: I was just thinking that maybe a more clear way to identify what this is could be:
"The following shows the output of the query using the operator specified above:"